stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -20,3 +20,28 @@ export async function getSecurity(securityId) {
   const security = await response.json();
   return security;
 }
+
+export async function postSecurity(security) {
+  const options = await getWindowOptions();
+  const response = await fetch(
+    `${options.customData.apiBaseUrl}/securities-v2`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(security)
+    }
+  );
+  if (response.status === 201) {
```

Comment:
When checking status codes it can be worth checking the group they fall in. To illustrate, all `2xx` responses are success type messages, `3xx` are *redirects*/*see other*, `4xx` are client errors and `5xx` are server errors. I haven't used the fetch API extensively but it may have some kind of helper methods to help you determine the type of response it received.

Reply:
I've changed this so that it checks if it lies between the upper and lower bands ie `>=200 && <=299 `